### PR TITLE
Add compiling-projects.rst and backtest-management.rst

### PR DESCRIPTION
**Pull request by GGoziker**

### CODE REVIEW

File: v2/our-platform/api-reference/backtest-management.rst [hunk 1]
```diff
@@ -0,0 +1,436 @@
...
+
+``POST`` /backtests/create
+
+Request
+=======
+
+.. code-block::
+
+    {
+      "projectId": 000000001,
```
Comment (jaredbroad) [inline v2/our-platform/api-reference/backtest-management.rst:139]: For projectId's use 12345678 as it doesn't have leading 0s.

For compileIds use "" strings as its a hash not a number

File: v2/our-platform/api-reference/backtest-management.rst [hunk 2]
```diff
@@ -0,0 +1,436 @@
...
+     - string
+     - Assigned backtest Id.
+   * - completed
+     - boolean
+     - True when the backtest is completed.
+   * - progress
+     - decimal
+     - Progress of the backtest in percent 0-1.
+   * - result
+     - A BacktestResult object (see full API reference for its attributes)
```
Comment (jaredbroad) [inline v2/our-platform/api-reference/backtest-management.rst:45]: Please add this object.

File: v2/our-platform/api-reference/backtest-management.rst [hunk 3]
```diff
@@ -0,0 +1,436 @@
...
+
+Returns the newly created Backtest object.
+
+.. code-block::
+
+    {
+      "name": "My first backtest",
+      "backtestId": "abc123",
+      "completed": false,
+      "progress": 0.5
```
Comment (jaredbroad) [inline v2/our-platform/api-reference/backtest-management.rst:171]: `"result": null,` for incomplete backtests

File: v2/our-platform/api-reference/compiling-projects.rst [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+``POST`` /compile/read
+
+Request
+=======
+
+.. code-block::
+
+    {
+      "projectId": 000000001,
+      "compileId": 12345
```
Comment (jaredbroad) [inline v2/our-platform/api-reference/compiling-projects.rst:109]: hash

File: v2/our-platform/api-reference/backtest-management.rst (hunk 2 above)
Comment (GGoziker) [reply v2/our-platform/api-reference/backtest-management.rst:45]: BacktestResult has a large, 19-class dependency chain, a big part of why the YAML grew to 2300 lines.

If we add BacktestResult, this RST file will also need to add:
- AlgorithmPerformance
  - TradeStatistics
  - PortfolioStatistics
  - Trade
    - TradeDirection
- AlphaRuntimeStatistics
   - InsightScore
- Chart
   - Series
    - ChartPoint
- Order
   - Symbol
   - SecurityType
   - OrderDirection
   - OrderSubmissionData
- OrderEvent
   - OrderStatus
   - OrderFee
   - CashAmount

Comment (jaredbroad) [reply v2/our-platform/api-reference/backtest-management.rst:45]: Correct. The API should be fully documented. Please add the needed objects.

**Pull request merged**